### ISSUE TITLE
Version bump

### ISSUE THREAD

**Issue by dmnkdmnt** [wayfair/git-parse#8]
Hey, could you please release a new version for the actual babel-polyfill / core-js 2 drop? :) Thanks

**Comment by rchl** [comment]
Will it ever happen?

**Comment by sodatea** [comment]
Ping @morganpackard

**Comment by BramKaashoek** [comment]
Please release the new version without babel-polyfill.

**Comment by morganpackard** [comment]
Hi folks. Apologies for the slow response. Are there specific pull requests you'd like to see merged in? If so, please let me know which ones.

**Comment by rchl** [comment]
@morganpackard see first post here. The PR was merged, it just wasn't released to npm.

**Comment by morganpackard** [comment]
Cool. Thanks. Will try to get to this quickly.

**Comment by morganpackard** [comment]
I'm getting test failures. Gonna be a bit before I can fix them. 
```
 SecurityError: localStorage is not available for opaque origins  
```

Pull requests which fix this issue are welcome.

**Comment by rchl** [comment]
When do you get that issue @morganpackard ? It doesn't seem like this project uses `localStorage` (at least not explicitly).

**Comment by dmnkdmnt** [comment]
The tests are not failing here - running node@10.16.3 / npm@6.9.0

**Comment by jchn-codes** [comment]
same for me, tests not failing node@10.16.3

**Comment by rchl** [comment]
They start failing after updating dependencies though. I've made PR that updates all dependencies and makes tests pass - https://github.com/wayfair/git-parse/pull/10

One note though - generated source code contains `'use strict'` clauses. Something that was previously on purpose avoided but I'm not sure why. With new babel `env` preset it's not that easy to remove those. But do we really need to?

**Comment by rchl** [comment]
All good after my PR was merged? Can we expect new version now?

**Comment by dmnkdmnt** [comment]
Is there any problem with publishing it? Don't want to rush, but I think @rchl is waiting for it since [2 Aug 2019](https://github.com/wayfair/git-parse/pull/1#issue-303735093).

**Comment by morganpackard** [comment]
Thanks for your patience all. Finally published `v1.0.4`.
